configure.ac

```diff
@@ -6,20 +6,20 @@ dnl # Run ./autogen.sh to build configure script
 dnl #
 dnl ##########################################################################
 
-AC_PREREQ(2.59)
+AC_PREREQ(2.63)
```

Comment:
Are we using specific macros from 2.63? If so it should be listed in the commit message.